app/src/main/java/com/example/healthinspector/Fragments/ScanFlow/RecommendProductsFragment.java

```diff
@@ -129,11 +136,25 @@ public void onResponse(JSONObject response) {
                                     keywordsBuilder.append(" ");
                                 }
                                 keywords = keywordsBuilder.toString();
+
+                                if(response.getJSONArray(PRODUCTS).getJSONObject(i).has(Constants.PRODUCT_IMAGE)){
```

Comment:
couple things: 
- we store `response.getJSONArray(PRODUCTS).getJSONObject(i)` in a variable to clean up code a little
- we can just get the key value and store it in imageUrl without the if check. If it's not there, it will just store null
- then if it is null, we can change it to empty string